### PR TITLE
Update comments and interface

### PR DESCRIPTION
**Pull request by gator-boi**
The purpose of this PR is to update some of the documentation in the contracts, and to rename the relayer reward methods to more intuitive names.

### CODE REVIEW

File: ethereum/contracts/coreRelayer/CoreRelayer.sol [hunk 1]
```diff
@@ -73,15 +75,15 @@ contract CoreRelayer is CoreRelayerGovernance {
         collectRelayerParameterPayment(relayParams, vm.emitterChainId, relayParams.deliveryGasLimit);
 
         RedeliveryInstructions memory redeliveryInstructions = RedeliveryInstructions({
-            payloadID: 3,
+            payloadID: uint8(3),
```
Comment (JoeHowarth) [inline ethereum/contracts/coreRelayer/CoreRelayer.sol:78]: Is this necessary?

File: ethereum/contracts/coreRelayer/CoreRelayer.sol [hunk 2]
```diff
@@ -288,7 +292,7 @@ contract CoreRelayer is CoreRelayerGovernance {
 
         // emit delivery status message
         DeliveryStatus memory status = DeliveryStatus({
-            payloadID: 2,
+            payloadID: uint8(2),
```
Comment (JoeHowarth) [inline ethereum/contracts/coreRelayer/CoreRelayer.sol:295]: ditto

**Review by JoeHowarth — APPROVED** [review]
👍

File: ethereum/contracts/coreRelayer/CoreRelayer.sol (hunk 2 above)
Comment (gator-boi) [reply ethereum/contracts/coreRelayer/CoreRelayer.sol:295]: The compiler will complain if that value is greater than type(uint8).max, so probably not necessary. I can remove them.

**Pull request merged**